tests/hedgehog/add_layer_to_allure.py

```diff
@@ -0,0 +1,80 @@
+import sys
+import argparse
+import yaml
+import json
+
+from os.path import exists
+from pathlib import Path
+
+
+def read_yaml(path):
+    with open(path, "r") as stream:
+        try:
+            return yaml.safe_load(stream)
+        except yaml.YAMLError as exc:
+            print(exc)
+
+
+def read_json(path):
+    with open(path, "r") as stream:
+        try:
+            return json.load(stream)
+        except json.JSONDecodeError as exc:
+            print(exc.msg)
+
+
+def add_layers(report_dir_path, test_to_layer_map, unmapped_test_path=None):
+    file_exists = exists(report_dir_path)
+    if not file_exists:
+        print("Error: Directory with Allure report doesn't exist.")
+        return -1
```

Comment:
just wondering, why not `exit(-1)` ? 
do we need to do something if `report_dir` is not exist?

Reply:
Currentlly `add_layers()` imports in **_tests/hedgehog_test_runner.py_**.  
I think some logic can be added after **_update_allure_**  step (in future)  and implicite `exit()`  will brake pipeline